docs/source/commands.rst

```diff
@@ -246,3 +246,12 @@ it's name or id could be used:
   gohan client subnet create --network "Network Name"
   gohan client subnet create --network network-id
   gohan client subnet create --network_id network-id
+
+Custom commands
+---------------
+
+Custom commands, if specified in schema, are also supported by gohan client and should be invoked as follows:
```

Comment:
Exactly that. user may not know that custom commands are generated from Custom Actions defined in https://github.com/cloudwan/gohan/blob/master/docs/source/schema.rst#custom-actions-schema